src/test/java/com/spotify/docker/client/DefaultDockerClientUnitTest.java

```diff
@@ -301,4 +306,80 @@ public void testBuildPassesMultipleRegistryConfigs() throws Exception {
     final JsonNode headerJsonNode = toJson(BaseEncoding.base64().decode(registryConfigHeader));
     assertThat(headerJsonNode, is(toJson(registryConfigs.configs())));
   }
+
+  @Test
+  public void testInspectNode() throws Exception {
+    final DefaultDockerClient dockerClient = new DefaultDockerClient(builder);
+
+    // build() calls /version to check what format of header to send
+    enqueueServerApiVersion("1.28");
```

Comment:
awesome 👏  although you don't have to repeat the comment each time you call this method